src/styles/stars.scss

```diff
@@ -0,0 +1,34 @@
+.reviews{
+  display: flex;
+  padding-left: 17px;
+  padding-right: 16px;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+
+  &__star {
+    display: inline;
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-position: left;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+}
+
+.reviews-text {
```

Comment:
Same here, it is a part of reviews inside of the card block. It is not a part of stars so you don't need to use it inside stars.scss